.github/workflows/redis-cluster.yml

```diff
@@ -0,0 +1,46 @@
+name: Redis Cluster - Install/Upgrade
+run-name: Redis Cluster - ${{ inputs.environment }} Redis Cluster in ${{ inputs.environment }} using ${{ github.ref_name }}
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment"
+        required: true
+        type: environment
+      action:
+        description: "Action"
+        required: true
+        type: choice
+        options:
+          - install
+          - upgrade
+
+jobs:
+  manageRedisCluster:
+    name: ${{ inputs.environment }} Redis Cluster in ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      NAMESPACE: ${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+    steps:
+      - name: Checkout Target Branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref_name }}
+      - name: Install CLI tools from OpenShift Mirror
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          oc: "4"
+      - name: Log in to OpenShift
+        run: |
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
+      - name: Manage Redis Cluster
+        working-directory: "./devops/helm/redis-cluster/"
+        run: |
+          if [ "${{ inputs.action }}" == "install" ]; then
+            make install NAMESPACE=${{ secrets.OPENSHIFT_ENV_NAMESPACE }}
+          elif [ "${{ inputs.action }}" == "upgrade" ]; then
+            PASSWORD=$(oc get secret redis-cluster -n ${{ secrets.OPENSHIFT_ENV_NAMESPACE }} -o jsonpath="{.data.redis-password}" | base64 --decode)
```

Comment:
Just double-checking, will it not print the Redis password to the GHA execution logs, right?

Reply:
What is the output of this one? Is there any change expected?

Reply:
https://www.aaron-powell.com/posts/2022-07-14-working-with-add-mask-and-github-actions/#using-add-mask

added in the below line

Reply:
I can see the `echo "::add-mask::$PASSWORD"` now in the file.
Sharing the GHA dos reference here for future reference: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions#masking-a-value-in-a-log